src/core/selectors.ts

```diff
@@ -0,0 +1,14 @@
+import { Selectors as FluxSelectors, Store } from '@storefront/flux-capacitor';
+import Configuration from './configuration';
+import ProductTransformer from './product-transformer';
+import { Structure } from './types';
+
+namespace Selectors {
+  export const transformProducts = (state: Store.State, products: Store.Product) => {
```

Comment:
I was think more like we could add selectors like `transformedProducts()`, `transformedAutocompleteProducts()` etc so that we're able to return a version of those selectors that automatically wraps the result in a call to `ProductTransformer.transform()`